node/src/chain_spec.rs

```diff
@@ -164,6 +165,21 @@ fn testnet_genesis(
 		"vesting": {
 			"vesting": Vec::<(AccountId, BlockNumber, BlockNumber, Balance)>::new(),
 		},
+		"claims": {
```

Comment:
Let's leave it empty here and provide a JSON example. Consider separate JSON snippet in the pallet's documentation or change in `qf-local.plain.json`.